workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts

```diff
@@ -65,15 +66,16 @@ export function setUpDynamicAuthorizationRoutes(router: Router, service: Dynamic
         });
         res.status(204).send();
       } catch (error) {
-        if (isPluginConfigurationError(error)) {
-          throw Boom.internal('An internal error occurred');
+        if (isGroupNotFoundError(error)) {
+          throw Boom.notFound('Role not found');
         }
         if (isUserNotFoundError(error)) {
-          throw Boom.tooManyRequests('User not found');
+          throw Boom.notFound('User not found');
         }
-        if (isRoleNotFoundError(error)) {
-          throw Boom.tooManyRequests('Role not found');
+        if (isPluginConfigurationError(error)) {
+          throw Boom.internal(error.message);
         }
```

Comment:
we dont need to check this here, `PluginConfigurationError`'s `IdpUnavailableError`'s and other internal errors are automatically turned into `Boom.internal` errors in the `unknownErrorHandler` function

Reply:
Removed.